docs/examples/util-crypto/04_generate_multisig_address/index.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable header/header */
+import { createKeyMulti, encodeAddress } from '@polkadot/util-crypto';
+
+function main () {
+  // Input the addresses that will make up the multisig account. This utility
```

Comment:
https://github.com/polkadot-js/common/blob/master/packages/util-crypto/src/address/sort.ts

Reply:
So with the above the comment is not "100%" correct, or rather, can show the used "how to sort" - since it needs to sort by the publicKey, not via ss58.